pkgs/modules/java/default.nix

```diff
@@ -1,19 +1,31 @@
-{ pkgs, lib, ... }:
+{ pkgs-unstable, lib, ... }:
```

Comment:
Why do we need to use unstable?

Reply:
The stable one didn't have version 19 of graalvm, which is needed to compile the project.

Reply:
Now that we switched to 23.05 probably don't need to.